dianna/dashboard/pages/Time_series.py

```diff
@@ -51,25 +60,75 @@
         contributing positively towards the classification decision are
         indicated in red and those who contribute negatively in blue.
         """)
+    elif load_example == "Scientific case: FRB":
+        ts_file = (data_directory / 'FRB211024.npy')
+        ts_model_file = (model_directory /
+                        'apertif_frb_dynamic_spectrum_model.onnx')
+        ts_label_file = (label_directory / 'apertif_frb_classes.txt')
+
+        # FRB data must be preprocessed
+        def preprocess(data):
+            """Preprocessing function for FRB use case to get the data in the right shape."""
+            return np.transpose(data, (0, 2, 1))[..., None].astype(np.float32)
+
+        ts_data = open_timeseries(ts_file)
+        ts_data_dianna = ts_data.T[None, ...]
+        ts_data_model = ts_data[None, ..., None]
```

Comment:
The variable names `ts_data_model` and `ts_data_dianna` are not informative, because the only difference between them is their shape; one is the transpose of the other. Could you rename these variables to something more descriptive?  Also, please add an inline comment explaining the purpose of defining these two variables.

Reply:
I renamed them to ts_data_predictor and ts_data_explainer respectively and added a comment to make it clearer